omim2obo/namespaces.py

```diff
@@ -59,7 +59,7 @@
 # LIDIA seems retired. so these are not resovable                   # Also: http://www.vetsci.usyd.edu.au/lida/
 LIDA = Namespace('http://sydney.edu.au/vetscience/lida/dogs/search/disorder/')  # Listing of Inherited Disorders in Animals (defunct?)
 OMIM = Namespace('https://omim.org/entry/')                                    # Online Mendelian Inheritance in Man (human disease and variants)
-OMIMPS = Namespace('https://www.omim.org/phenotypicSeries/PS')                   # Online Mendelian Inheritance in Man (phenotypes)
+OMIMPS = Namespace('https://omim.org/phenotypicSeries/PS')                   # Online Mendelian Inheritance in Man (phenotypes)
```

Comment:
Want me to check that / make sure that the rest of the `www` deletion has been taken care of of, per?:
- #103

Reply:
Yes, we need a PR on mondo-ingest to that end as well! Thank you!